tests/phpunit/ServicesTests/UserRoleServiceTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\ServicesTests;
```

Comment:
The other suite was marked as `ControllerTests` where as this one is `ServicesTests`

They should both be singular (i.e. `ServiceTests`)

Reply:
gotcha